pkg/distro/fedora/images.go

```diff
@@ -368,6 +368,31 @@ func diskImage(workload workload.Workload,
 	return img, nil
 }
 
+func archiveImage(workload workload.Workload,
```

Comment:
In rhel this is called `TarImage`, lets be consistent here (c.f. https://github.com/osbuild/images/blob/main/pkg/distro/rhel/images.go#L791 ) and call it the same (either here or in the other place). Note that those will be unified hopefully not too far in the future and having similar names will make this easier.

Reply:
Done.